pkgs/applications/virtualization/OVMF/default.nix

```diff
@@ -2,25 +2,47 @@
 , csmSupport ? false, seabios ? null
 , secureBoot ? false
 , httpSupport ? false
+, crossIa32 ? false
 }:
 
 assert csmSupport -> seabios != null;
+assert crossIa32 == false || (stdenv.isi686 || stdenv.isx86_64);
```

Comment:
```suggestion
```

Reply:
This is only to ensure someone doesn't crossIa32 on AARCH or something.